docs/advanced/production-ready/media-middleware.mdx

```diff
@@ -355,7 +355,8 @@ could be implemented:
 
 ```js
 import { Router } from "express";
-import proxy from "express-http-proxy";
+// import proxy from "express-http-proxy"; for < 2.21
```

Comment:
I let this here as I thought that developers may use this page before updating to the latest FC version.

**question:** do you think it makes sense? should we do it for the invoices page too?